networkx/algorithms/bipartite/tests/test_centrality.py

```diff
@@ -6,7 +6,7 @@ class TestBipartiteCentrality(object):
 
     def setUp(self):
         self.P4 = nx.path_graph(4)
-        self.K3 = nx.complete_bipartite_graph(3,3)
+        self.K3 = nx.bipartite.complete_bipartite_graph(3,3)
```

Comment:
Isn't `complete_bipartite_graph` in the main namespace. Then it should be `self.K3 = nx.complete_bipartite_graph(3,3)`